tap_github/client.py

```diff
@@ -197,10 +197,16 @@ def validate_response(self, response: requests.Response) -> None:
             # The GitHub API randomly returns 401 Unauthorized errors, so we try again.
             if (
                 response.status_code == 401
-                and "unauthorized" in str(response.content).lower()
+                and "unauthorized" in str(response.reason).lower()
```

Comment:
how about changing that to: `and not "bad credentials" in str(response.content).lower()` and remove the changes below?

Also, it seems that "content" is more informative than "reason", should we change the messages to this?

```
f"{response.status_code} Client Error: "
f"{response.content} for path: {full_path}"
```

Reply:
or both content and reason?